src/components/inputs/text/ValidationInput/ValidationInput.tsx

```diff
@@ -18,6 +18,7 @@ function ValidationInput({
   setValue,
   setIsValid,
   validationPredicates,
+  onClick,
```

Comment:
nit: default this value to `() => return`

Reply:
added